dungeon/src/contrib/hud/UIUtils.java

```diff
@@ -64,30 +64,74 @@ public static void center(final Actor actor) {
   }
 
   /**
-   * Creates line breaks after a word once a certain character count is reached.
+   * This is a simple implementation of formatting a text with soft word wrap.
+   *
+   * <p>Characteristics:
+   *
+   * <ul>
+   *   <li>Every space character (i.e. {@code " "} or {@code "\n"}, etc.) that occurs at least once
+   *       should be replaced by exactly one space character.
+   *   <li>A line of text should not be longer than {@code MAX_ROW_LENGTH} (currently 40)
+   *       characters.
+   *   <li>Long words at the end of a line should be inserted into the next line (soft word wrap).
+   *   <li>The {@code '\n'} character should be used as the line separator.
```

Comment:
ist es wirklich `\n` oder muss hier rücksicht auf auf das betriebssystem genommen werden? also brauchen wir evtl. `System.getProperty("line.separator")`?

Reply:
libgdx kommt mit nur-`\n` als zeilenumbruchzeichen unter windows zurecht, dann nehme ich an, dass das unter linux, wo es ja `"\r\n"` nicht gibt, auch so ist. also nein, es müsste keine rücksicht auf das betriebssystem genommen werden.

zur sicherheit kann ich es aber auch noch einmal testen.

Reply:
ja, bitte mal ausprobieren. und hier als erklärung mit aufnehmen, weil in einem jahr bestimmt wieder irgendwer darüber stolpert.